koku/koku/metrics.py

```diff
@@ -39,95 +35,12 @@ def connection_check(self):
             LOG.error("DatabaseStatus.connection_check: No connection to DB: %s", str(error))
             DB_CONNECTION_ERRORS_COUNTER.inc()
 
-    def query(self, query, query_tag):
```

Comment:
This `DatabaseStatus` class exists only to collect this schema size metric, so we ought to remove the entire thing + the `collect_metrics` celery task.